From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Wed, 30 Nov 2022 09:59:33 +0100
Subject: docs: Use main branch instead of master

The old name still works but let's use the current name.

Origin: upstream, 3.24.36, commit:785e2ab168f025e8c4bf0250100827caffb60ba5
---
 docs/reference/gtk/resources.sgml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/reference/gtk/resources.sgml b/docs/reference/gtk/resources.sgml
index e51e382..eeca69a 100644
--- a/docs/reference/gtk/resources.sgml
+++ b/docs/reference/gtk/resources.sgml
@@ -51,7 +51,7 @@ questions will be directed to Discourse.
 
 <para>
 Please, follow the instructions outlined in the
-<ulink url="https://gitlab.gnome.org/GNOME/gtk/-/blob/master/CONTRIBUTING.md">Contribution Guide</ulink> available in the GTK
+<ulink url="https://gitlab.gnome.org/GNOME/gtk/-/blob/main/CONTRIBUTING.md">Contribution Guide</ulink> available in the GTK
 source code repository.
 </para>
 
